projects/go-lib/src/lib/components/go-accordion/go-accordion.component.ts

```diff
@@ -47,6 +47,7 @@ export class GoAccordionComponent implements OnInit, AfterContentInit {
       // NOTE: This feels a little destructive.
       // We lose track of the icon explicitly set by the child component.
       panel.icon = this.showIcons ? panel.icon : null;
+      panel.updateClasses();
```

Comment:
The minor trade off is that we have to manually trigger the updateClass function because this method of manually setting a child's attributes doesn't fire lifecycle events.